ToDoApp/ToDoApp/ToDoServiceImpl.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ToDoServiceImpl.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-23.
+//
+
+import Foundation
+import FirebaseDatabase
+import FirebaseDatabaseSwift
+enum TaskError: Error {
```

Comment:
is this error type still used?

Reply:
No, it's no longer used, I'll delete in the next commit